components/mpas-seaice/cime_config/buildnml

```diff
@@ -243,8 +243,8 @@ def buildnml(case, caseroot, compname):
             grid_prefix = 'mpassi.SOwISC12to60E2r4.rstFromG-anvil'
 
     elif ice_grid == 'ECwISC30to60E2r1':
-        grid_date = '210413'
-        grid_prefix = 'seaice.ECwISC30to60E2r1'
+        grid_date = '230314'
+        grid_prefix = 'partitions/seaice.ECwISC30to60E2r1'
```

Comment:
@xylar these changes should be `decomp_*`, rather than `grid_*`.

Reply:
Woops, thanks!